Controller/AuthorizeController.php

```diff
@@ -126,11 +129,11 @@ protected function getRedirectionUrl(UserInterface $user)
      */
     protected function getClient()
     {
+        $request = $this->container->get('request_stack')->getCurrentRequest();
```

Comment:
does not work on Symfony 2.3. Please keep a fallback on the ``request`` service when there is no RequestStack``.

and this should be done inside the ``if``